pkg/controller.v1/pytorch/initcontainer.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2021 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+package pytorch
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/yaml"
+
+	pytorchv1 "github.com/kubeflow/training-operator/pkg/apis/pytorch/v1"
+	"github.com/kubeflow/training-operator/pkg/config"
+)
+
+var (
+	initContainerTemplate = `
+- name: init-pytorch
+  image: {{.InitContainerImage}}
+  imagePullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 100m
```

Comment:
For this kind of init container, is it better to remove the resource requests?

Reply:
It is to be compatible with openshift https://github.com/kubeflow/pytorch-operator/pull/276